ui/src/app/dashboard/grouped-summary/grouped-summary.component.ts

```diff
@@ -1,49 +1,130 @@
-import {Component, Input} from '@angular/core';
+import {Component, Input, OnInit} from '@angular/core';
 import {JobStatus} from "../../shared/model/JobStatus";
 import {AggregationEntry} from "../../shared/model/AggregationEntry";
 import {Aggregation} from "../../shared/model/Aggregation";
 import {ActivatedRoute} from "@angular/router";
 import {URLSearchParamsUtils} from "../../shared/utils/url-search-params.utils";
+import {JobStatusIcon} from "../../shared/common";
+import {Sort} from "@angular/material";
+
+const LABEL_KEY = 'label';
+const DEFAULT_NUM_ROW = 5;
 
 @Component({
   selector: 'jm-grouped-summary',
   templateUrl: './grouped-summary.component.html',
   styleUrls: ['./grouped-summary.component.css']
 })
-export class GroupedSummaryComponent {
+export class GroupedSummaryComponent implements OnInit{
   @Input() aggregation: Aggregation;
   @Input() statusArray: Array<JobStatus>;
+  aggregationEntries = [];
+  unsortedAggregationEntries = [];
+
+  labelKey = LABEL_KEY;
+  expand = false;
+
+  displayCollapseArrow = false;
+  iconTransform = 'rotate(180deg)';
+  paddingBottom = '';
```

Comment:
Not sure I like plumbing a style value this way. I would probably either apply a class to the card depending on whether or not it is expanded - or else set an inline style in the template based on `expanded`

Reply:
Accepted.